src/migration/MigrationExecutor.ts

```diff
@@ -1,15 +1,25 @@
 import {Table} from "../schema-builder/table/Table";
+import {TableColumn} from "../schema-builder/table/TableColumn";
 import {Connection} from "../connection/Connection";
 import {Migration} from "./Migration";
+import {MigrationInterface} from "./MigrationInterface";
 import {ObjectLiteral} from "../common/ObjectLiteral";
+import {CollectionUtils} from "../util/CollectionUtils";
 import {PromiseUtils} from "../util/PromiseUtils";
 import {QueryRunner} from "../query-runner/QueryRunner";
 import {SqlServerDriver} from "../driver/sqlserver/SqlServerDriver";
 import {MssqlParameter} from "../driver/sqlserver/MssqlParameter";
 import {SqlServerConnectionOptions} from "../driver/sqlserver/SqlServerConnectionOptions";
 import {PostgresConnectionOptions} from "../driver/postgres/PostgresConnectionOptions";
-import { MongoDriver } from "../driver/mongodb/MongoDriver";
-import { MongoQueryRunner } from "../driver/mongodb/MongoQueryRunner";
+import {MongoDriver} from "../driver/mongodb/MongoDriver";
+import {MongoQueryRunner} from "../driver/mongodb/MongoQueryRunner";
+import {sha1} from "object-hash";
```

Comment:
Currently by design we should import all external dependencies via `PlatformTools.load` to make it possible to use library in browser without webpack (something that is going to change in the future, but we need it for now).

Reply:
Thanks, didn't know about that. Should be fixed!